samples/openthread/cli/harness/nRF_Connect_SDK.py

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env python
 #
-# Copyright (c) 2020 Nordic Semiconductor ASA
+# Copyright (c) 2021 Nordic Semiconductor ASA
```

Comment:
This change is not needed since it only states the date of the creation of the file.

Reply:
I thought file modification resets the copyrights timeout but thanks for clarification. Will push the change in a moment.